python/lsst/ts/m2/csc.py

```diff
@@ -129,7 +130,9 @@ def __init__(
         self._port_telemetry = port_telemetry
```

Comment:
Somewhere here in the init file, consider setting the version of ts-m2comm in the `softwareVersions` event.

```
from lsst.ts.m2comm import __version__ as __m2comm_version__
```

```
self.evt_softwareVersions.set(subsystemVersions=f"ts-m2comm={__m2comm_version__}")
```

Reply:
Done in the commit: [2ce50af](https://github.com/lsst-ts/ts_m2/pull/28/commits/2ce50af7b174ca88c63068bb94e185ac8586ddb9). Thanks!